### PR TITLE
Add account sensitized

### PR DESCRIPTION
**Pull request by noellabo**
This is a moderation function.

If your target account does not properly flag sensitive when attaching sensitive media attachments or posting with URLs, you can flag your account as sensitive instead of silence or suspend.

- Add a sensitive flag to the activities that local users deliver to remote. It cannot be updated and cannot be canceled.
- The remote and local user status retains the original sensitive flags, but this moderation overwrites them dynamically. When you remove the sensitive flag from your account, it follows the original status sensitive flag.

### CODE REVIEW

**Comment by Sasha-Sorokin** [comment]
Are you sure that “sensitized” is a good word choice, it's probably okay in the code, but as for the user it is certainly confusing. Maybe just say that “account is marked as sensitive” or something like this?

**Comment by noellabo** [comment]
Thanks Sasha-Sorokin.
I fixed it, but I'm not sure if it's a proper expression.

File: config/locales/en.yml [hunk 1]
```diff
@@ -1300,11 +1308,13 @@ en:
       subject:
         disable: Your account %{acct} has been frozen
         none: Warning for %{acct}
+        sensitive: Your account %{acct} marked as sensive
         silence: Your account %{acct} has been limited
         suspend: Your account %{acct} has been suspended
       title:
         disable: Account frozen
         none: Warning
+        sensitive: Account is marked as sensive
```
Comment (Gargron) [inline config/locales/en.yml:1317]: Typo: `sensive` -> `sensitive`

I am a little concerned about the choice of words. This is consistent with "Mark media as sensitive" but "sensitive" has a different meaning when talking about people: "easily hurt or damaged"

Comment (noellabo) [reply config/locales/en.yml:1317]: Oops!

How about this notation?
> Mark the media and URLs that your account posts as sensitive

**Comment by noellabo** [comment]
Fixed conflicts and changed wording.

**Review by Gargron — APPROVED** [review]

**Comment by Kjwon15** [comment]
Please add this type of moderation on domain range too

**Pull request merged**